js/flatbuffers.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+
+import { flatbuffers } from "flatbuffers";
+import * as util from "./util";
+
+const globalBuilder = new flatbuffers.Builder();
+
+// Re-export some types.
+export type Offset = flatbuffers.Offset;
+export class ByteBuffer extends flatbuffers.ByteBuffer {}
+export class Builder extends flatbuffers.Builder {}
+
+// This is a wrapper around the real Builder .
+// The purpose is to reuse a single ArrayBuffer for every message.
+// We can do this because the "control" messages sent to the privileged
+// side are guarenteed to be used during the call to libdeno.send().
+export function createBuilder(): Builder {
+  let bb = globalBuilder.dataBuffer();
```

Comment:
You should somehow assert that the global buffer is not in use before reusing it.

Reply:
Fixed.